app/grandchallenge/cases/serializers.py

```diff
@@ -208,3 +213,74 @@ def _get_linked_task(*, targets):
         )
     else:
         raise RuntimeError(f"Unknown target {targets=}")
+
+
+class CSImageSerializer(serializers.BaseSerializer):
+    """
+    Sserializes a cases.Image object into a Cornerstone image object as
+    defined in https://docs.cornerstonejs.org/api.html#image.
+    """
```

Comment:
If this goes to production we shouldn't serve this in this way, the images should be pre-processed by some background task and stored for serving directly from the CDN. Could you add a bit TODO explaining this and that this function is behind a feature flag?

Reply:
The more I think about it I really think this should be moved client-side, could you do this in JS instead?